framework/wazuh/core/cluster/common.py

```diff
@@ -1685,6 +1685,7 @@ def send_data_to_wdb(data, timeout, info_type='agent-info'):
     result : dict
         Dict containing number of updated chunks, error messages (if any) and time spent.
     """
+    ignored_wdb_exceptions = ['Cannot execute Global database query; FOREIGN KEY constraint failed']
```

Comment:
Since it's a single constant message, I think it would be beneficial to move it outside of the function and use a global variable. This way, we would avoid the (minimal but avoidable) footprint of initializing a list on every call to `send_data_to_wdb()`.